interpreter/cling/lib/Interpreter/DynamicLibraryManager.cpp

```diff
@@ -59,26 +59,128 @@ namespace cling {
       addSearchPath(P, /*IsUser*/ false);
   }
 
+  ///\returns substitution of pattern in the front of original with replacement
+  /// Example: substFront("@rpath/abc", "@rpath/", "/tmp") -> "/tmp/abc"
+  static std::string substFront(llvm::StringRef original, std::string pattern,
```

Comment:
Why not
```suggestion
  static std::string substFront(llvm::StringRef original, llvm::StringRef pattern,
```
? Looks like https://github.com/root-project/root/pull/6969#discussion_r618397300 wasn't applied properly, or I'm missing rationale.